src/current/cockroachcloud/billing-management.md

```diff
@@ -61,4 +61,6 @@ Under the **Spend over time** section, you can see a table of each of your organ
 
 ## View invoices
 
-You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period.
+You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period. 
```

Comment:
@Amruta-Ranade one call out here - users can't actually "download" invoices. I realize this was probably already written like this but worth correcting. They can see the full invoice in the UI but they can't actually download it.

Reply:
Fixed!